imports/plugins/included/surcharges/server/no-meteor/util/getSurchargeMessageInShopLanguage.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @name getSurchargeMessageInShopLanguage
+ * @summary Returns translated surcharge message
+ * @param {String} language Language to filter items by
+ * @param {Array} messagesByLanguage Surcharge messages by language array
+ * @return {String} Translated surcharge message, or first surcharge message if translation in
+ *  provided language is not available.
+ */
+export default function getSurchargeMessageInShopLanguage(language, messagesByLanguage) {
```

Comment:
I would remove "Shop" from this function name. It's not the shop language; it's whatever language the client asked for in the query.